models/repo.go

```diff
@@ -929,11 +944,12 @@ func MigrateRepository(doer, u *User, opts MigrateRepoOptions) (*Repository, err
 
 	migrateTimeout := time.Duration(setting.Git.Timeout.Migrate) * time.Second
```

Comment:
Should this timeout be changed from its current 600 seconds?

Reply:
You can change this on app.ini